src/app/pages/observation-list/image-list/image-list.component.html

```diff
@@ -5,15 +5,18 @@
   </ion-refresher>
 
   <div class="list-header">
-    <p>Viser alle bilder fra alle filtrerte observasjoner.</p>
+    <p>{{ "OBSERVATION_LIST.IMAGES_DESCRIPTION" | translate }}</p>
```

Comment:
Regobs.no har ingen overskrift her. Tror kanskje vi kan vurdere å droppe den eller endre selve hoved-tittelen for sida i stedet fra "Observasjoner" til "Bilder fra observasjoner"?
![image](https://github.com/user-attachments/assets/890c1d4f-c668-4745-a709-e2d51b225b25)

Reply:
Enig med Marcin, det kan eventuelt tas i en egen oppgave.